native/cocos/network/HttpClient.h

```diff
@@ -31,6 +31,7 @@
 #include <condition_variable>
 #include <thread>
 #include "base/RefVector.h"
+#include "base/ThreadPool.h"
```

Comment:
Remove this line , include it in cpp file.

Reply:
done